components/gitpod-db/src/typeorm/entity/db-blocked-repository.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { BlockedRepository } from "@gitpod/gitpod-protocol/src/blocked-repositories-protocol";
+import { Entity, Column, CreateDateColumn, DeleteDateColumn, UpdateDateColumn, PrimaryGeneratedColumn } from "typeorm";
+
+@Entity()
+export class DBBlockedRepository implements BlockedRepository {
+    @PrimaryGeneratedColumn()
+    id: number;
+
+    @Column("varchar")
+    urlRegexp: string;
+
+    @Column()
+    blockUser: boolean;
+
+    @CreateDateColumn()
+    createdAt: string;
+
+    @UpdateDateColumn()
+    updatedAt: string;
+
+    @DeleteDateColumn()
+    deletedAt: string;
+
+    // TODO: not sure if we need this?
+    // TODO: should have a default value of false?
+    // This column triggers the db-sync deletion mechanism. It's not intended for public consumption.
+    @Column()
+    deleted: boolean;
```

Comment:
As the comment suggests, I'm not sure if we need this field on the new table.

Reply:
@andrew-farries For this DB table to be synced we also need an entry in `tables.ts`. Happy to chat about how that needs to look like.

Reply:
I've added an entry to `tables.ts` in c4aebe35ccc2540fc03e4ef8f547e56a8be1a652